monitoring/grafana/dashboards/pool-overview.json

```diff
@@ -15,223 +15,713 @@
   "editable": false,
   "gnetId": null,
   "graphTooltip": 0,
-  "iteration": 1551789900270,
+  "id": 15,
```

Comment:
@pcuzner @epuertat This is breaking the `ceph dashboard grafana dashboards update` command because by adding this id to the dashboard, the command fails because the id is not null and the dashboard doesn't exist.

```console
fatal: [mon0 -> mon0]: FAILED! => {
    "changed": false,
    "cmd": [
        "podman",
        "exec",
        "ceph-mon-mon0",
        "ceph",
        "--cluster",
        "ceph",
        "dashboard",
        "grafana",
        "dashboards",
        "update"
    ],
    "delta": "0:00:01.087702",
    "end": "2021-05-20 17:53:59.140587",
    "rc": 22,
    "start": "2021-05-20 17:53:58.052885"
}

STDERR:

Error EINVAL: Traceback (most recent call last):
  File "/usr/share/ceph/mgr/mgr_module.py", line 1335, in _handle_command
    return self.handle_command(inbuf, cmd)
  File "/usr/share/ceph/mgr/dashboard/module.py", line 417, in handle_command
    push_local_dashboards()
  File "/usr/share/ceph/mgr/dashboard/grafana.py", line 136, in push_local_dashboards
    retry()
  File "/usr/share/ceph/mgr/dashboard/grafana.py", line 93, in __call__
    result = self.func(*self.args, **self.kwargs)
  File "/usr/share/ceph/mgr/dashboard/grafana.py", line 131, in push
    grafana.push_dashboard(body)
  File "/usr/share/ceph/mgr/dashboard/grafana.py", line 58, in push_dashboard
    response.raise_for_status()
  File "/lib/python3.6/site-packages/requests/models.py", line 940, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://192.168.15.10:3000/api/dashboards/db
```

The 404 http code isn't on the API endpoint but on the dashboard ressouce which returns:

`{'message': 'Dashboard not found', 'status': 'not-found'}`

If we remove this `id` line (like it was before) then the command works again.

https://grafana.com/docs/grafana/latest/http_api/dashboard/#create--update-dashboard

Reply:
@dsavineau mmm, ok, I think the issue here is that we didn't assume that this would have to work with Ceph-ansible, as since the advent of cephadm, the `grafana dashboards update` is no longer required (the Grafana dashboard are pre-built in the Grafana image)... :/ but given the fix is easy, let's go with it.

Reply:
And, of course, thanks for letting us know!